SpinCore_pp/ppg/echo.py

```diff
@@ -27,8 +27,8 @@ def run_spin_echo(
     SW_kHz,
     output_name,
     indirect_fields=None,
-    ph1_cyc=r_[0, 1, 2, 3],
-    ph2_cyc=r_[0],
+    ph1_cyc=psp.r_[0, 1, 2, 3],
+    ph2_cyc=psp.r_[0],
```

Comment:
this is not good b/c r_ comes from numpy.  Also I usually do `from numpy import r_` since its point is as a convenience function.